clientv3/watch.go

```diff
@@ -72,6 +72,9 @@ type Watcher interface {
 
 	// Close closes the watcher and cancels all watch requests.
 	Close() error
+
+	// RequestProgress requests a progress notify response be sent in all watch channels.
+	RequestProgress(ctx context.Context) error
```

Comment:
Move this above Close method? Also rename this to just Progress?

Reply:
Actually, after second thought RequestProgress is better.

Reply:
Thanks @xiang90, I've moved above `Close()`.